doku/tests/printers/enum/of_tag/adjacent/of_fields/named/mod.rs

```diff
@@ -74,4 +74,14 @@ printer_test! {
     "output.with-user-comments-wrapped.separated.json" => to_json_fmt(WithUserCommentsWrapped, {
          "enums_style": "Separated",
     }),
+
+    "output.without_key_quotes.with-user-comments-wrapped.commented.json" => to_json_fmt(WithUserCommentsWrapped, {
```

Comment:
nit: it looks like in here we usually use `-` to separate words (e.g. `with-user-comments`); would you mind renaming `without_key_quotes` into `without-key-quotes`? 👀